src/components/TopNavigationBar/TopNavigationBar.tsx

```diff
@@ -182,9 +181,12 @@ export default function TopNavigationBar({
           <div className="max-w-screen-xl mx-auto py-3 px-3 sm:px-6 lg:px-8">
             <div className="pr-16 sm:text-center sm:px-16">
               <p className="font-medium text-white">
-                <span className="md:hidden">Register for the USACO January Workshop!</span>
+                <span className="md:hidden">
+                  Register for the USACO January Workshop!
+                </span>
                 <span className="hidden md:inline">
-                  USACO Bronze/Silver solutions explained in a live workshop TODAY!
+                  USACO Bronze/Silver solutions explained in a live workshop
+                  TODAY!
```

Comment:
this alr passed right?

Reply:
wha

Reply:
how did this get in there

Reply:
alr i pulled from master, this should fix it